src/client/automation/get-element.js

```diff
@@ -8,14 +8,16 @@ var positionUtils = testCafeCore.positionUtils;
 var domUtils      = testCafeCore.domUtils;
 
 
-function getElementFromPoint (x, y) {
-    testCafeUI.hide();
+function getElementFromPoint (x, y, underTopRoot) {
```

Comment:
`underTopRoot` is not unclear for me. At first look I didn't get that it's about shadowUI root.
Maybe 'underTopShadowUIElement' or something like this?